client/src/components/Search/Search.js

```diff
@@ -267,6 +269,27 @@ class Search extends Component {
     setTimeout(() => this.focusFirstFormField(selector), 50);
   }
 
+  onClickOut(evt) {
+    // This is a fairly rudimentory way to detect if the user clicked on an entirely different
+    // DOM element, which probably uses jQuery
+    // It's important that we only call show() when a relevent element is clicked, otherwise we'll
+    // end up making lots of unnecessary AJAX requests to the form schema endpoint
+    const buttonEl = evt.target;
+    const $componentEl = window.jQuery(this.nodeRef.container);
+    const filterButtonClicked = buttonEl.matches('#filters-button, .font-icon-search, .filter-open');
+    if (filterButtonClicked && $componentEl.is(':visible')) {
```

Comment:
```suggestion
    // The visibility check is needed to avoid loading the form for hidden tabs (e.g. in ModelAdmin)
    if (filterButtonClicked && $componentEl.is(':visible')) {
```

Reply:
Updated